xbmc/input/linux/LinuxInputDevices.cpp

```diff
@@ -370,7 +370,7 @@ int CLinuxInputDevice::KeyboardGetSymbol(unsigned short value)
     case 0x7f:
       return XBMCK_BACKSPACE;
     case 0xa4:
-      return XBMCK_EURO; /* euro currency sign */
+      return 0x20ac; /* euro currency sign */
```

Comment:
should this be updated in xbmc/input/XBMC_keysym.h instead ?

Reply:
Certainly possible (I actually considered this), but since input of the Euro sign already worked flawlessly on other platforms (tested on desktop Linux), I decided not to mess with it to avoid possible side-effects.

Reply:
+1 for coding it here. 0x20ac is a unicode value, not a XBMC Key.